ddtrace/tracer/sqlcomment.go

```diff
@@ -57,6 +57,8 @@ const (
 	sqlCommentDBService     = "dddbs"
 	sqlCommentParentVersion = "ddpv"
 	sqlCommentEnv           = "dde"
+	sqlCommentPeerHostname  = "ddh"
+	sqlCommentPeerDBName    = "dddb"
```

Comment:
It's a bit unclear to me what `Peer` means in this context. Why do these have `Peer` but not the others? What makes them different?

Reply:
Added a comment explaining